compute_test.go

```diff
@@ -130,37 +133,49 @@ func TestGetAndExec(t *testing.T) {
 
 	runnable := fns[0]
 
-	result, err := client.ExecString(runnable, "world")
+	result, uuid, err := client.ExecString(runnable, "world")
 	if err != nil {
 		t.Fatal(err)
 	}
 
 	t.Log(string(result))
 
+	// Give the results a moment to propagate
+	time.Sleep(1 * time.Second)
```

Comment:
lets mock time in future PRs, time.Sleep is never the answer

Reply:
This is maybe the _one_ time it's the answer, since this is real time (these tests are full integration tests against a live instance).